mariadb/content.md

```diff
@@ -171,9 +183,13 @@ The `-v /my/own/datadir:/var/lib/mysql` part of the command mounts the `/my/own/
 
 If there is no database initialized when the container starts, then a default database will be created. While this is the expected behavior, this means that it will not accept incoming connections until such initialization completes. This may cause issues when using automation tools, such as `docker-compose`, which start several containers simultaneously.
 
+## Health/Liveness/Readiness Checking
+
+See [the "Official Images" FAQ](https://github.com/docker-library/faq#healthcheck) for why there isn't a `HEALTHCHECK` directive. There is however a `/usr/local/bin/healthcheck.sh` script that provides a variety of options for what you considers health/liveness/readiness. This is documented within the script itself.
+
 ## Usage against an existing database
 
-If you start your `%%IMAGE%%` container instance with a data directory that already contains a database (specifically, a `mysql` subdirectory), the `$MARIADB_ROOT_PASSWORD` variable should be omitted from the run command line; it will in any case be ignored, and the pre-existing database will not be changed in any way.
+If you start your `%%IMAGE%%` container instance with a data directory that already contains a database (specifically, a `mysql` subdirectory), no environment variables that control initialization will be needed or examined, and the pre-existing database will not be changed in any way. The only environment variable that has an effect if the existing data directory is populated is `MARIADB_AUTO_UPGRADE`.
```

Comment:
```suggestion
If you start your `%%IMAGE%%` container instance with a data directory that already contains a database (specifically, a `mysql` subdirectory), no environment variables that control initialization will be needed or examined, and no pre-existing databases will be changed. The only exception is the non-default `MARIADB_AUTO_UPGRADE` environment variable, that might cause `mysql_upgrade`/`mariadb-upgrade` to run, which might indeed update your databases, especially the `mysql` database.
```

Reply:
Good.
"your databases" isn't quite right as its "--upgrade-system-tables" - https://github.com/MariaDB/mariadb-docker/blob/774a77c330aeb10a8b1ffee99d54e9f6821b5d79/docker-entrypoint.sh#L406 so I'll probably word around system tables instead of `mysql` database.